dotCMS/src/integration-test/java/com/dotcms/MainSuite.java

```diff
@@ -64,11 +55,7 @@
 import com.dotcms.publishing.manifest.ManifestReaderFactoryTest;
 import com.dotcms.rendering.velocity.directive.DotParseTest;
 import com.dotcms.rendering.velocity.servlet.VelocityServletIntegrationTest;
-import com.dotcms.rendering.velocity.viewtools.DotTemplateToolTest;
-import com.dotcms.rendering.velocity.viewtools.FileToolTest;
-import com.dotcms.rendering.velocity.viewtools.JSONToolTest;
-import com.dotcms.rendering.velocity.viewtools.MessageToolTest;
-import com.dotcms.rendering.velocity.viewtools.XmlToolTest;
+import com.dotcms.rendering.velocity.viewtools.*;
```

Comment:
As a general rule, we should not use wildcard imports.

Reply:
Done!